config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -13,22 +13,31 @@ periodics:
     preset-e2e-gci-gce-scalability: "true"
   spec:
     containers:
-    - args:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20181205-915278e90-master
```

Comment:
At the high level - we would like to migrate all 100-node clusters imho.
There are a bunch of those (IIRC, thing like beta, stable1, stable2, maybe even more...)